gui/main.py

```diff
@@ -1,4 +1,6 @@
-
```

Comment:
I've seen this done before in other projects where a file of just imports is created and then that file is imported to make the importing on these main files cleaner. Definitely not necessary, but what would you all think of that?